src/compiler/scala/reflect/reify/phases/Calculate.scala

```diff
@@ -26,7 +26,7 @@ trait Calculate {
     }
 
   /**
-   *  Merely traverses the reifiee and records symbols local to the reifee along with their metalevels.
+   *  Merely traverses the reifier and records symbols local to the reifee along with their metalevels.
```

Comment:
I did mean the "reifee" here as in "something that is being reified".

Clearly, the term is cryptic enough to confuse readers, so I'd suggest replacing it with "the tree that is being reified"

Reply:
perhaps "target"

Reply:
Do you mean you'd like me to change "reifier" to "reifee"  for now ?
I'm not sure I understand it completely, but I can only say that "reifiee" is typo.

Reply:
sorry..,I can not understand.
I am waiting for your instructions.

Reply:
I'm suggesting "Merely traverses the target", Eugene suggested "Merely traverses the tree being reified". Either suggestion seems acceptable to me